type_data/migrations/39_breakables_noncorp.rb

```diff
@@ -0,0 +1,169 @@
+create_table "breakable", keys: [:name]
+create_table "noncorporeal", keys: [:prefix, :name, :exclude]
+
+migrate "breakable" do
+  insert(:name, %{large clump of barnacles})
+  insert(:name, %{slimy thick-shelled carapace})
+  insert(:name, %{ashy lump of charcoal})
+  insert(:name, %{thick clump of debris})
+  insert(:name, %{small pile of old debris})
+  insert(:name, %{solid clod of dirt})
+  insert(:name, %{vile clump of excrement})
+  insert(:name, %{foul pile of excrement})
+  insert(:name, %{hunk of rotting flesh})
+  insert(:name, %{dusty glass jar})
+  insert(:name, %{yellowed glass jar})
+  insert(:name, %{putrid pile of muck})
+  insert(:name, %{large brittle rock})
+  insert(:name, %{large crumbly rock})
+  insert(:name, %{hollowed petrified root})
+  insert(:name, %{withered black rose})
+  insert(:name, %{hardened clump of sand})
+  insert(:name, %{large piece of sandstone})
+  insert(:name, %{handful of congealed green slime})
+  insert(:name, %{handful of congealed slime})
+  insert(:name, %{hardened dark chitin})
+end
+
+migrate "noncorporeal" do
```

Comment:
In this or a follow up you should consider going through the migrations to see if there is anything else we should add. Off the top of my head these two would definitely be good to add: https://github.com/elanthia-online/scripts/blob/master/type_data/migrations/36_necrotic_ring_boss.rb and https://github.com/elanthia-online/scripts/blob/master/type_data/migrations/8_reim.rb

Reply:
Sounds good. I went through them all but got held up on two. The AotA from EG and whether any of those creatures are non-corp: https://github.com/elanthia-online/scripts/blob/master/type_data/migrations/35_eg_arena_of_the_abyss.rb
And if I missed anything from OSA https://github.com/elanthia-online/scripts/blob/master/type_data/migrations/33_osa_kf_npcs.rb as I'm not sure if "The Ethereal Captain" is undead as well as I don't have it marked as such. Also if "unworldly" is undead and non-corp.